compendium/tools/SCRCodeGen/test/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+#-----------------------------------------------------------------------------
+# Build and run the GTest Suite of tests
+#-----------------------------------------------------------------------------
+
+set(scrcodegen_test_exe_name SCRCodeGenTests)
+
+include_directories(
+  ${GTEST_INCLUDE_DIRS}
+  ${GMOCK_INCLUDE_DIRS}
+  )
+
+
+if (US_COMPILER_CLANG OR US_COMPILER_APPLE_CLANG)
```

Comment:
don't think this if statement is needed if its already in the top level CMakeLists.txt

Reply:
Done.